cloud-sql/mysql/Startup.cs

```diff
@@ -144,19 +145,19 @@ DbConnection NewMysqlUnixSocketConnection()
             // [START cloud_sql_mysql_dotnet_ado_connection_socket]
             // Equivalent connection string: 
             // "Server=<dbSocketDir>/<INSTANCE_CONNECTION_NAME>;Uid=<DB_USER>;Pwd=<DB_PASS>;Database=<DB_NAME>;Protocol=unix"
-            String dbSocketDir = Environment.GetEnvironmentVariable("DB_SOCKET_PATH") ?? "/cloudsql";
-            String instanceConnectionName = Environment.GetEnvironmentVariable("INSTANCE_CONNECTION_NAME");
+            String dbSocketDir = Environment.GetEnvironmentVariable("DB_SOCKET_PATH") ?? "/cloudsql"; 

```

Comment:
Nit: remove the extra white space that was added at the end of the line, and in the next line as well. Or maybe you were trying to add example values here as well